website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/schema/schema-metadata-models.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Models"
+id: "metadata-schema-models"
+---
+
+import { ArgsTable, SchemaTable } from "./schema";
+
+
+The models object allows you to query information about many models in a given job.
+
+### Arguments
+
+When querying for `models`, the following arguments are available:
+
+<ArgsTable queryName="models" />
```

Comment:
can we clarify that the `database`, `schema`, and `identifier` fields are all case-insensitive? I'm ok with that living in the API spec for sure